Musico/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+    before_action :check_login, except: [:login, :new, :validate , :create]
```

Comment:
@mohseengs dont use except in base controllers, use skip_before_action instead in the respective controller where you have the actions.